### PR TITLE
Use proper previous tag

### PR DESCRIPTION
**Pull request by Thodor12**

### CODE REVIEW

File: gradle/mod.gradle [hunk 1]
```diff
@@ -1161,6 +1161,22 @@ tasks.withType(Jar.class, {task ->
     task.duplicatesStrategy 'exclude'
 })
 
+tasks.register("outputChangelogHeader") {
+    def changelog = new java.io.File("changelog.md")
```
Comment (Nightenom) [inline gradle/mod.gradle:1165]: I really have feeling this should be "file" but Thodor insisted
```suggestion
    def changelog = file("changelog.md")
```

**Review by Nightenom — APPROVED** [review]
Seen results, was generally fine, but at Orion: when merging please use squash commit

File: gradle/mod.gradle (hunk 1 above)
Comment (marchermans) [reply gradle/mod.gradle:1165]: This needs to indeed be file.

**Review by marchermans — CHANGES_REQUESTED** [review]
Use gradle features.

File: gradle/mod.gradle (hunk 1 above)
Comment (Thodor12) [reply gradle/mod.gradle:1165]: Maybe useful if you state the reason. This needs to adhere to cwd, not project root relative.

Comment (marchermans) [reply gradle/mod.gradle]: No actually, i needs to adhere to the project directory.
That is a core concept within gradle projects.

**Review by marchermans — APPROVED** [review]

**Pull request merged**